.github/runs-on.yml

```diff
@@ -0,0 +1,8 @@
+runners:
+  self-hosted-linux:
+    cpu: [8, 16]
+    ram: [16, 32, 64]
```

Comment:
I'm afraid it may not fit on 16G in some cases, like when building releases. Our self-hosted runners were 32G, weren't they?

Similarly 8 vCPU will probably be quite slow, how CI times between current self-hosted runners and these compare?

Reply:
I guess we can use bigger specs now that we are running spot instances.